.github/workflows/helm-release.yml

```diff
@@ -8,9 +8,6 @@ env:
   CR_PACKAGE_PATH: "${{ github.workspace }}/.cr-release-packages"
   CR_TOOL_PATH: "${{ github.workspace }}/.cr-tool"
   CR_VERSION: "1.5.0"
-permissions:
-  contents: read
-  id-token: write
```

Comment:
don't you need the `id-token` for the github app token to work?

Reply:
No, it works without. A v0.0.1 release was built.

Reply:
I am using the `actions/create-github-app-token@v1` action. That creates a token from the github app app-ID and private key.